[debug] [2023-12-12T13:29:44.716Z] ----------------------------------------------------------------------
[debug] [2023-12-12T13:29:44.717Z] Command:       /usr/local/bin/firebase /Users/matsumotoshouhi/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/bin/firebase serve
[debug] [2023-12-12T13:29:44.717Z] CLI Version:   12.9.1
[debug] [2023-12-12T13:29:44.718Z] Platform:      darwin
[debug] [2023-12-12T13:29:44.718Z] Node Version:  v16.16.0
[debug] [2023-12-12T13:29:44.720Z] Time:          Tue Dec 12 2023 22:29:44 GMT+0900 (GMT+09:00)
[debug] [2023-12-12T13:29:44.720Z] ----------------------------------------------------------------------
[debug] 
[debug] [2023-12-12T13:29:44.855Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2023-12-12T13:29:44.856Z] > authorizing via signed-in user (221205157@ccmailg.meijo-u.ac.jp)
[debug] [2023-12-12T13:29:44.857Z] [iam] checking project test for permissions ["firebase.projects.get"]
[debug] [2023-12-12T13:29:44.859Z] >>> [apiv2][query] POST https://cloudresourcemanager.googleapis.com/v1/projects/test:testIamPermissions [none]
[debug] [2023-12-12T13:29:44.860Z] >>> [apiv2][(partial)header] POST https://cloudresourcemanager.googleapis.com/v1/projects/test:testIamPermissions x-goog-quota-user=projects/test
[debug] [2023-12-12T13:29:44.860Z] >>> [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/test:testIamPermissions {"permissions":["firebase.projects.get"]}
[debug] [2023-12-12T13:29:48.628Z] <<< [apiv2][status] POST https://cloudresourcemanager.googleapis.com/v1/projects/test:testIamPermissions 200
[debug] [2023-12-12T13:29:48.630Z] <<< [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/test:testIamPermissions {}
[debug] [2023-12-12T13:29:48.632Z] [iam] error while checking permissions, command may fail: FirebaseError: Authorization failed. This account is missing the following required permissions on project test:

  firebase.projects.get
[debug] [2023-12-12T13:29:48.633Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects/test [none]
[debug] [2023-12-12T13:30:01.861Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects/test 403
[debug] [2023-12-12T13:30:01.867Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects/test {"error":{"code":403,"message":"The caller does not have permission","status":"PERMISSION_DENIED"}}
[debug] [2023-12-12T13:30:01.870Z] HTTP Error: 403, The caller does not have permission
[debug] [2023-12-12T13:30:06.895Z] Error: Timed out.
    at Timeout._onTimeout (/Users/matsumotoshouhi/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/utils.js:254:49)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
[error] 
[error] Error: An unexpected error has occurred.
[debug] [2023-12-12T13:31:00.086Z] ----------------------------------------------------------------------
[debug] [2023-12-12T13:31:00.087Z] Command:       /usr/local/bin/firebase /Users/matsumotoshouhi/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/bin/firebase deploy --only hosting:deploytest-5fcd2
[debug] [2023-12-12T13:31:00.088Z] CLI Version:   12.9.1
[debug] [2023-12-12T13:31:00.088Z] Platform:      darwin
[debug] [2023-12-12T13:31:00.088Z] Node Version:  v16.16.0
[debug] [2023-12-12T13:31:00.092Z] Time:          Tue Dec 12 2023 22:31:00 GMT+0900 (GMT+09:00)
[debug] [2023-12-12T13:31:00.092Z] ----------------------------------------------------------------------
[debug] 
[debug] [2023-12-12T13:31:00.201Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2023-12-12T13:31:00.202Z] > authorizing via signed-in user (221205157@ccmailg.meijo-u.ac.jp)
[debug] [2023-12-12T13:31:00.203Z] [iam] checking project test for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2023-12-12T13:31:00.205Z] >>> [apiv2][query] POST https://cloudresourcemanager.googleapis.com/v1/projects/test:testIamPermissions [none]
[debug] [2023-12-12T13:31:00.206Z] >>> [apiv2][(partial)header] POST https://cloudresourcemanager.googleapis.com/v1/projects/test:testIamPermissions x-goog-quota-user=projects/test
[debug] [2023-12-12T13:31:00.207Z] >>> [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/test:testIamPermissions {"permissions":["firebase.projects.get","firebasehosting.sites.update"]}
[debug] [2023-12-12T13:31:01.145Z] <<< [apiv2][status] POST https://cloudresourcemanager.googleapis.com/v1/projects/test:testIamPermissions 200
[debug] [2023-12-12T13:31:01.148Z] <<< [apiv2][body] POST https://cloudresourcemanager.googleapis.com/v1/projects/test:testIamPermissions {}
[debug] [2023-12-12T13:31:01.148Z] [iam] error while checking permissions, command may fail: FirebaseError: Authorization failed. This account is missing the following required permissions on project test:

  firebase.projects.get
  firebasehosting.sites.update
[debug] [2023-12-12T13:31:01.149Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects/test [none]
[debug] [2023-12-12T13:31:01.649Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects/test 403
[debug] [2023-12-12T13:31:01.650Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects/test {"error":{"code":403,"message":"The caller does not have permission","status":"PERMISSION_DENIED"}}
[debug] [2023-12-12T13:31:01.651Z] HTTP Error: 403, The caller does not have permission
[debug] [2023-12-12T13:31:06.664Z] Error: Timed out.
    at Timeout._onTimeout (/Users/matsumotoshouhi/.cache/firebase/tools/lib/node_modules/firebase-tools/lib/utils.js:254:49)
    at listOnTimeout (node:internal/timers:559:17)
    at processTimers (node:internal/timers:502:7)
[error] 
[error] Error: An unexpected error has occurred.
